Utils/Dataflow/pyDKB/dataflow/stage/AbstractProcessorStage.py

```diff
@@ -311,13 +305,16 @@ def stream_input(self, fd):
         Split stream into messages;
         Yield Message object.
         """
-        if self.__EOMessage == '\n':
+        if self.ARGS.eom == '\n':
             iterator = iter(fd.readline, "")
-            for line in iterator:
-                yield self.parseMessage(line)
         else:
-            raise NotImplementedError("Stream input is not implemented for"
-                                      " custom EOMessage marker.")
+            iterator = custom_readline(fd, self.ARGS.eom)
+        try:
+            for i, line in enumerate(iterator):
```

Comment:
Do we really need `i` here? Won\`t simple `for line in iterator` also work?

Reply:
@mgolosova 
It was created for printing the line number in a previous variant of KeyboardInterrupt exception handler. I forgot to turn this line back into the previous form. I'll fix it. Thank you!